### ISSUE TITLE
error: processInner: not implemented for type: *ast.FuncLit

### ISSUE THREAD

**Issue by Splizard** [ghostiam/protogetter#10]
Fails to scan function literals.

**Comment by ghostiam** [comment]
Please provide an example of the code that is causing the problem.

**Comment by Splizard** [comment]
It's something like this:

```go
type proto.Message struct {
	Name string
	Value string
}

var items []*proto.Message

value := items[slices.IndexFunc(items, func(message *proto.Message) bool {
	return strings.Contains(strings.ToLower(message.GetName()), "specific value")
})].Value,
```

**Comment by ghostiam** [comment]
Fixed in version 0.3.6.
Please wait for golangci-lint update.
